helpers/traversal.go

```diff
@@ -5,10 +5,21 @@ import (
 
 	"github.com/ipfs/go-fetcher"
 	"github.com/ipld/go-ipld-prime"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
 	"github.com/ipld/go-ipld-prime/traversal/selector"
 	"github.com/ipld/go-ipld-prime/traversal/selector/builder"
 )
 
+var matchAllSelector ipld.Node
+
+func init() {
+	ssb := builder.NewSelectorSpecBuilder(basicnode.Prototype.Any)
+	matchAllSelector = ssb.ExploreRecursive(selector.RecursionLimitNone(), ssb.ExploreUnion(
+		ssb.Matcher(),
+		ssb.ExploreAll(ssb.ExploreRecursiveEdge()),
+	)).Node()
+}
```

Comment:
AFAICT this selector means `matchAll` and there's no reason to rebuild it continuously given that it's a static thing that shouldn't eat much memory.

Reply:
FWIW, I also added one of these upstream recently, here: https://github.com/ipld/go-ipld-prime/blob/df94e6a99727d6dfd9f3381f0f407f3df368178a/traversal/selector/parse/selector_parse.go#L94

(Though I'm fine seeing a construction snippet like this recur locally in places, as needed, too.  I added it to upstream as much for an example as for any other purpose; I don't know if it actually really "belongs" in the API surface area up there.)